src/main/java/com/sprint/mission/discodeit/security/CustomUserDetails.java

```diff
@@ -0,0 +1,51 @@
+package com.sprint.mission.discodeit.security;
+
+import com.sprint.mission.discodeit.entity.User;
+import java.util.Collection;
+import java.util.Collections;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+@RequiredArgsConstructor
+public class CustomUserDetails implements UserDetails {
+
+  private final User user;
+
+  @Override
+  public Collection<? extends GrantedAuthority> getAuthorities() {
+    return Collections.singleton(new SimpleGrantedAuthority("ROLE_USER"));
```

Comment:
`ROLE_USER`만 반환하면 롤 계층 구조가 정상적으로 동작하지 않을 것 같아요.  

로그인한 사용자에게 hasRole() 호출 시 `ADMIN`, `CHANNEL_MANAGER` 도 일반 사용자로 인식하기에, `ADMIN`, `CHANNEL_MANAGER`만 접근가능한 메서드가 정사동작하지 않을 확률이 높을 것 같습니다.  

아래처럼 바꿔보는건 어떨까요?
```suggestion
    return Collections.singleton(new SimpleGrantedAuthority("ROLE_" + user.getRole().name()));
```